lib/openlogi/shipment.rb

```diff
@@ -0,0 +1,27 @@
+require "openlogi/base_object"
+
+module Openlogi
+  class Shipment < BaseObject
+    attribute :id, String
+    attribute :identifier, String
+    attribute :created_at, DateTime
```

Comment:
I missed a response. Certainly it is. `id` and `created_at` are for system value so that the document does not have them maybe.